### PR TITLE
Post AUC beta run dplyr lesson updates

### LINKED ISSUE #160: When demonstrating the OR operator, change the code chunk to use two numerical variables. Is currently filter(variants, sample_id == "SRR2584863", (INDEL | QUAL >= 100)) and INDEL is logical.

### PR DESCRIPTION
**Pull request by ytakemon**
Currently, we filter on INDEL which is a logical column: `filter(variants, sample_id == "SRR2584863", (INDEL | QUAL >= 100))` and INDEL is logical. Instead of INDEL, we will now filter on MQ: `filter(variants, sample_id == "SRR2584863", (MQ >= 50 | QUAL >= 100))`

### CODE REVIEW

**Comment by ytakemon** [comment]
Did not know you could "draft" a pull request to indicate it is not yet ready for merging! Very cool!

**Comment by ytakemon** [comment]
All issues listed under "Data wrangling and analysis with tidyverse"  in our mega issue: #142 has been addressed and is now ready for review. Pinging @JasonJWilliamsNY @naupaka and @poterlowicz-lab (whoever has a moment) to review and merge.

**Review by naupaka — CHANGES_REQUESTED** [review]
Nice work @ytakemon! I spotted one minor thing to fix.

File: _episodes_rmd/05-dplyr.Rmd [hunk 1]
```diff
@@ -52,20 +52,33 @@ then load it to be able to use it.
...
+library("tidyr")          ## loads in tidyr package to use
+library("ggplot2")          ## loads in ggplot2 package to use
 library("readr")          ## load in readr package to use
 ```
 
 You only need to install a package once per computer, but you need to load it
 every time you open a new R session and want to use that package.
 
+> ## Tip: Installing packages
+> It may be temping to install the `tidyverse` packages, as it contains many
```
[the remark this answers is not in the document]
Comment (ytakemon) [reply _episodes_rmd/05-dplyr.Rmd]: Fixed in [5925ee7](https://github.com/datacarpentry/genomics-r-intro/pull/161/commits/5925ee76d57e8f0b36e3fb08063e9f770f5d1ef5)

**Review by naupaka — APPROVED** [review]

**Pull request merged**